From: Evgeni Golov <evgeni@debian.org>
Subject: install files to /usr/lib instead of libexec
 Debian has no /usr/libexec
Forwarded: no
diff --git a/Makefile b/Makefile
index 16049d6..78130f0 100644
--- a/Makefile
+++ b/Makefile
@@ -195,7 +195,7 @@ install: install-dirs
 	# libexec scripts
 	$(foreach file, $(wildcard libexec/*), \
 		$(call install_python_script, \
-			$(file), $(DESTDIR)/usr/libexec/tuned/$(notdir $(file))))
+			$(file), $(DESTDIR)/usr/lib/tuned/$(notdir $(file))))
 
 	# icon
 	install -Dpm 0644 icons/tuned.svg $(DESTDIR)$(DATADIR)/icons/hicolor/scalable/apps/tuned.svg
