diff --git a/tuned-adm.py b/tuned-adm.py
index 1b9623d..1df8cc3 100755
--- a/tuned-adm.py
+++ b/tuned-adm.py
@@ -100,7 +100,11 @@ if __name__ == "__main__":
 	debug = options.pop("debug")
 	asynco = options.pop("async")
 	timeout = options.pop("timeout")
-	action_name = options.pop("action")
+	try:
+		action_name = options.pop("action")
+	except KeyError:
+		parser.print_usage(file = sys.stderr)
+		sys.exit(1)
 	log_level = options.pop("loglevel")
 	result = False
 
